src/Cardano/Types/Transaction.purs

```diff
@@ -198,6 +198,7 @@ newtype TxBody = TxBody
   , auxiliaryDataHash :: Maybe AuxiliaryDataHash
   , validityStartInterval :: Maybe Slot
   , mint :: Maybe Mint
+  , referenceInputs :: Maybe (Array TransactionInput)
```

Comment:
Should we use `Set` instead? 
```
 /**
   * @param {TransactionInputs} reference_inputs
   */
  set_reference_inputs(reference_inputs: TransactionInputs): void;
```
There's probably an enforced ordering.

Reply:
I'm not actually certain on this admittedly.

Reply:
I'd prefer to preserve the ordering, it's safer to be closer to CTL than not. We don't have `Set` as API requirement, right?